### DISCUSSION TITLE
Carla Sim HELP!

### DISCUSSION THREAD

**Discussion by hAl0G3n98** [community/community#126959] [Programming Help]
### Body

Hello. I am looking for someone to help me with carla simulator. I need to make a simulation with 2 cars in an intersection, 1 autonomous and 1 normal. The normal car should cut off the face of the autonomous vehicle, so i can analyze data from the sensors. After that i want to be able to control the autonomous car by myself, to compare. Can someone help me with this request?
Thank you!

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebndev** [comment]
Hi @hAl0G3n98,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
